packages/harmony/src/components/progress-bar/ProgressBar.tsx

```diff
@@ -14,18 +11,23 @@ const getBN = (num: ProgressValue): BN => {
   return new BN(num)
 }
 
-export const ProgressBar: React.FC<ProgressBarProps> = ({
-  className,
-  sliderClassName,
-  sliderBarClassName,
-  min = new BN(0),
-  max = new BN(100),
-  value,
-  showLabels = false,
-  minWrapper: MinWrapper,
-  maxWrapper: MaxWrapper,
-  ...other
-}: ProgressBarProps) => {
+function clampBN(value: BN, min: BN, max: BN): BN {
+  return BN.min(BN.max(value, min), max)
+}
+
+export const ProgressBar = (props: ProgressBarProps) => {
```

Comment:
Are these "harmony approved" components now or are we considering these just "legacy stems ports"? If its the latter, can we add a comment explaining that they're still the same as the old stems ones?

Reply:
Yeah great call, will add beta to all the descriptions